internal/cmd/rsl/log/log.go

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package log
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/gittuf/gittuf/internal/repository"
+	"github.com/spf13/cobra"
+)
+
+type options struct{}
+
+func (o *options) AddFlags(_ *cobra.Command) {}
+
+func (o *options) Run(_ *cobra.Command, _ []string) error {
+	repo, err := repository.LoadRepository()
+	if err != nil {
+		return err
+	}
+	entries, annotationMap, err := repository.LogRSLEntries(repo)
+	if err != nil {
+		return err
+	}
+	log := ""
+
+	for index := len(entries) - 1; index >= 0; index-- {
+		entry := entries[index]
+
+		log += fmt.Sprintf("entry %v", entry.ID)
+
+		skipped := false
+
+		if annotations, ok := annotationMap[entry.ID]; ok {
+			for _, annotation := range annotations {
+				if annotation.Skip {
+					skipped = true
+					break
+				}
+			}
+		}
+
+		if skipped {
+			log += " (Skipped Entry)\n"
+		} else {
+			log += "\n"
+		}
+
+		if annotations, ok := annotationMap[entry.ID]; ok {
+			for _, annotation := range annotations {
+				log += fmt.Sprintf("    Message:\n        %s\n", annotation.Message)
+			}
+		}
+
+		log += "\n"
+	}
+
+	cmd := exec.Command("echo", log)
+	pager := exec.Command("less")
+
+	pager.Stdin, _ = cmd.StdoutPipe()
+	pager.Stdout = os.Stdout
+	pager.Stderr = os.Stderr
+
+	if err := pager.Start(); err != nil {
+		return fmt.Errorf("error: %s", err.Error())
+	}
+	if err := cmd.Run(); err != nil {
+		return fmt.Errorf("error: %s", err.Error())
+	}
+	if err := pager.Wait(); err != nil {
+		return fmt.Errorf("error: %s", err.Error())
+	}
+	return nil
+}
+
+func New() *cobra.Command {
+	o := &options{}
+	cmd := &cobra.Command{
+		Use:               "log",
+		Short:             "Display the RSL entry log",
```

Comment:
I think this doesn't line up with the generated docs. Also since the L in RSL is log, I wonder if more tweaking is in order.

Reply:
I have changed it to "Displays the Reference State Log"